pymc/model.py

```diff
@@ -988,23 +989,28 @@ def dim_lengths(self) -> Dict[str, Variable]:
 
     @property
     def test_point(self) -> Dict[str, np.ndarray]:
-        """Deprecated alias for `Model.initial_point(seed=None)`."""
+        """Deprecated alias for `Model.initial_point(random_seed=None)`."""
         warnings.warn(
-            "`Model.test_point` has been deprecated. Use `Model.initial_point(seed=None)`.",
+            "`Model.test_point` has been deprecated. Use `Model.initial_point(random_seed=None)`.",
             FutureWarning,
         )
         return self.initial_point()
 
-    def initial_point(self, seed=None) -> Dict[str, np.ndarray]:
+    def initial_point(self, random_seed: SeedSequenceSeed = None) -> Dict[str, np.ndarray]:
         """Computes the initial point of the model.
 
+        Parameters
+        ----------
+        random_seed : SeedSequenceSeed, default None
+            Seed(s) for generating initial point from the model. Used in pymc.aesaraf.reseed_rngs
```

Comment:
Not sure if this is the correct indication of this type in the docstring. Kind of long created type. i.e. 
https://github.com/pymc-devs/pymc/blob/5d7283ea581266e05ca88ce2123d71bcba5f7c8e/pymc/aesaraf.py#L1021

Also mentioned the function that this value is fed into. Not sure how that renders atm.

Reply:
Looks about right (from [here](https://pymcio--6291.org.readthedocs.build/projects/docs/en/6291/api/generated/classmethods/pymc.Model.initial_point.html#pymc-model-initial-point))

![grafik](https://user-images.githubusercontent.com/5894642/201720338-32fb1237-d0e9-45b0-b9f8-43ffba129297.png)